### ISSUE TITLE
req vs Req

### ISSUE THREAD

**Issue by kyleabeauchamp** [ParmEd/ParmEd#91]
There's a bit of inconsistency with capitalization in parameters:

```
In [37]: prmtop.bond_types[0]
Out[37]: <BondType; k=570.000, Req=1.229>



In [38]: prmtop.bond_types[0].Req
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-38-c483e0741d7f> in <module>()
----> 1 prmtop.bond_types[0].Req

AttributeError: 'BondType' object has no attribute 'Req'

In [39]: prmtop.bond_types[0].req
Out[39]: 1.229
```

**Comment by swails** [comment]
Good catch.  Should be fixed for everything in `topologyobjects` by #92.

Thanks!

**Comment by swails** [comment]
Fixed in #92. `__repr__` printouts for all topology and parameter objects were updated to be consistent with the actual attribute names.
